modules/ROOT/pages/x12-receive-read-settings.adoc

```diff
@@ -1,148 +1,162 @@
+a|Sends a functional acknowledgment (the receipt and syntactical acceptability of a message) when there is a failure to route the message to one of the message flows.
+
+
 |Disabled
 |===
 
 === X12 997 Functional Acknowledgment
 
-The X12 997 functional acknowledgment reports the status of a received interchange. It reports all the errors encountered while processing the received document.
-
+The X12 997 functional acknowledgment reports the status of a received interchange. It reports all of the errors encountered while processing the received document.
 As with all X12 transaction sets, the 997 functional acknowledgment is sent inside a GS or GE envelope.
 
-The 997 functional acknowledgment is used to:
+The 997 functional acknowledgment does the following:
 
-* Acknowledge the receipt of an interchange or functional group.
-* Accept or reject one or more functional groups or transactions.
-* Verify and report compliance with the standard.
+* Acknowledges the receipt of an interchange or functional group
+* Accepts or rejects one or more functional groups or transactions
+* Verifies and reports compliance with the standard
 
-If an interchange contains multiple groups, then a 997 acknowledgment is returned for each group.
+If an interchange contains multiple groups, a 997 acknowledgment is returned for each group.
 
-If a group contains multiple transaction sets, then the acknowledgment for that group contains multiple AK2 loops, that is, one for each transaction set. This applies only if the AK2 loops are generated for accepted transaction sets.
+If a group contains multiple transaction sets, then the acknowledgment for that group contains one loop for each transaction set. This applies only if AK2 loops are generated for accepted transaction sets.
 
 [[validation-settings]]
 == Validation Settings
 
-Selecting the checkbox (enabling) for any of these options means that all the X12 documents meet this condition criteria and receive a 997 failure message:
+Use these settings to ensure that all X12 messages received from a partner meet validation rules. Parter Manager logs all validation errors and reports them in the CONTRL acknowledgments.
+*Vijay, is it correct to say "the host rejects", or should we say "Partner Manager rejects"?
 
 [%header%autowidth.spread]
 |===
-|Setting |Description |Default
+|Setting a|Description |Default
 
-|Fail document when value length outside allowed range
-|If selected, a transaction with values that are too long or too short is rejected. If unselected, the values are used and the transaction is not rejected. In either case, the error is logged and reported in the 997 acknowledgments.
-|Enabled
+|Fail when value length outside allowed range
+a| * If selected, the host rejects transactions with values that are too long or too short.
```

Comment:
@dhirsch1 - Let's phrase it as below:
If selected, Partner Manager rejects transactions with data element values that are too long or too short against the definition in the EDI schema used.